src/styles/settings/_breakpoints.scss

```diff
@@ -1,3 +1,4 @@
 $breakpoint-large-screen-860: 54rem;
 $breakpoint-grid-large-screen-1115: 72rem;
 $breakpoint-scale-up: 51rem;
+$breakpoint-specification-block-grid: 59.375rem; // 950px
```

Comment:
I'd suggest renaming this to `$breakpoint-specifications-grid-two-column`, since it tells you a little more about what happens at that width.

Reply:
These two changes are all set--here's the commit: [d8048e7](https://github.com/sparkbox/accessible-components/pull/99/commits/d8048e78b0849412f4ebff4bb1736c1f77d9b9b0)

Reply:
* and nitpicky is good -- being better is always good! :)